### PR TITLE
[zh] Sync 1.18 change notes and upgrade notes into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Translate 1.18 change notes and upgrade notes into Chinese
```
content/zh/news/releases/1.18.x/_index.md
content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md
content/zh/news/releases/1.18.x/announcing-1.18/upgrade-notes/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+- **修复** 修复了 Gateway API 中 Gateway 资源的 Programmed 条件上报问题。
+  ([Issue #43498](https://github.com/istio/istio/issues/43498))
+
+- **修复** 修复了当在具有相同端口和不同协议的网关中指定不同的绑定时，无法正确生成侦听器的问题。
+  ([Issue #43688](https://github.com/istio/istio/issues/43688))
+
+- **修复** 修复了当在具有相同端口和 TCP 协议的网关中指定不同的绑定时，无法正确生成侦听器的问题。
+  ([Issue #43775](https://github.com/istio/istio/issues/43775))
+
+- **修复** 修复了在某些情况下删除服务入口时不会删除相应端点的问题。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md:61]:
```suggestion
- **修复** 修复了在某些情况下删除 ServiceEntry 时不会删除相应端点的问题。
```

File: content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+  ([Issue #43853](https://github.com/istio/istio/issues/43853))
+
+- **修复** 修复了自动分配的服务入口 IP 在主机重用时会发生变化的问题。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md:64]:
```suggestion
- **修复** 修复了自动分配的 ServiceEntry IP 在主机重用时会发生变化的问题。
```

File: content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+  ([Issue #43440](https://github.com/istio/istio/issues/43440))
+
+- **修复** 修复了使用自动注册的虚拟机时会忽略没有在 `WorkloadGroup` 中定义的标签的问题。
+  ([Issue #32210](https://github.com/istio/istio/issues/32210))
+
+- **升级** 升级了 gateway-api 集成以读取 `ReferenceGrant`、`Gateway`
+  和 `GatewayClass` 的 `v1beta1` 版本资源。用户在升级 Istio 之前，需要保证
+  gateway-api 须高于 `v0.6.0+` 版。`istioctl x precheck` 可以在升级前检测到这个问题。
+
+- **移除** 移除了针对 Kubernetes `Gateway` Pod 中 `proxy.istio.io/config` 注释的支持。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md:117]:
```suggestion
- **移除** 移除了针对 Kubernetes `Gateway` Pod 中 `proxy.istio.io/config` 注解的支持。
```

File: content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+
+- **移除** 移除了对 `Ingress` `networking.k8s.io/v1beta1` 版本的支持。
+  其 `v1` 版本从 Kubernetes 1.19 版开始可用。
+
+- **移除** 默认移除了 Gateway API `alpha` 版类型。可以使用
+  `PILOT_ENABLE_ALPHA_GATEWAY_API=true` 来显式的重新启用它们。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md:123]:
```suggestion
  `PILOT_ENABLE_ALPHA_GATEWAY_API=true` 来显式地重新启用这些类型。
```

File: content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+
+## 安全性 {#security}
+
+- **新增** 新增了 `--profiling` 标志以允许在 pilot-agent 状态端口上启用或禁用分析能力。
+  ([Issue #41457](https://github.com/istio/istio/issues/41457))
+
+- **新增** 新增了对将额外的联邦信任域从 `caCertificates` 推送到对等 SAN 验证器的支持。
+  ([Issue #41666](https://github.com/istio/istio/issues/41666))
+
+- **新增** 新增了在使用 ECDSA 时对 P384 曲线的支持
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md:144]:
```suggestion
- **新增** 新增了在使用 ECDSA 时对 P384 曲线的支持。
```

File: content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+- **新增** 新增了可以自动将 GOMEMLIMIT 设置为 `istiod`，用于降低发生内存不足所带来的风险。
+  ([Issue #40676](https://github.com/istio/istio/issues/40676))
+
+- **新增** 新增了支持通过 `.Values.labels` 将标签添加到网关 Pod 模板中的能力。
+  ([Issue #41057](https://github.com/istio/istio/issues/41057))
+  ([Issue #43585](https://github.com/istio/istio/issues/43585))
+
+- **新增** 新增了通过验证 `.Values.pilot.env.EXTERNAL_CA` 和
+  `.Values.global.pilotCertProvider` 参数来限制外部 CA `usecases` 的
+  k8s CSR 权限的 `clusterrole` 检查能力。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md:235]:
```suggestion
  K8s CSR 权限的 `clusterrole` 检查能力。
```

File: content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+- **新增** 新增了 `bug-report` 中的 istiod 指标，以及一些如 `telemetry` 之类的调试点。
+  ([Issue #44062](https://github.com/istio/istio/issues/44062))
+
+- **新增** 在 `istioctl pc route` 的输出中新增了一个“VHOST NAME”列。
+  ([Issue #44413](https://github.com/istio/istio/issues/44413))
+
+- **新增** 为不同的 `istioctl dashboard` 命令添加了本地标志 `--ui-port`
+  以允许用户指定用于仪表板的 UI 组件端口。
+
+- **修复** 修复了 Server Side Apply 应在 1.22 以上的 Kubernetes 版集群中默认启用，
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md:291]:
```suggestion
- **修复** 修复了 Server Side Apply 应在 1.22 版本以上的 Kubernetes 集群中默认启用，
```

File: content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md [hunk 8]
```diff
@@ -0,0 +1,344 @@
...
+  `istioctl manifests generate --set <boolvar>=<bool>`
+  错误地将布尔值转换为字符串的问题。
+  ([Issue #43355](https://github.com/istio/istio/issues/43355))
+
+- **修复** 修复了 `istioctl experimental describe` 在 VirtualService
+  被定义为跨多个服务拆分流量时不显示所有加权路由的问题。
+  ([Issue #43368](https://github.com/istio/istio/issues/43368))
+
+- **修复** 修复了 `istioctl x precheck` 命令会显示由 Istio
+  默认设置的且不需要的 IST0136 消息的问题。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/change-notes/index.md:304]:
```suggestion
  默认设置的但实际不需要的 IST0136 消息的问题。
```

File: content/zh/news/releases/1.18.x/announcing-1.18/upgrade-notes/index.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+## Gateway API 自动部署变更 {#gateway-api-automated-deployment-changes}
+
+仅当您使用 [Gateway API 自动部署](/zh/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment)功能时，
+此变更才会对您产生影响。请注意，这仅适用于 Kubernetes Gateway API，
+不适用于 Istio `Gateway`。您可以使用以下命令检查您是否正在使用该功能：
+
+    {{< text bash >}}
+    $ kubectl get gateways.gateway.networking.k8s.io -ojson | jq -r '.items[] | select(.spec.gatewayClassName == "istio") | select((.spec.addresses | length) == 0) | "Found managed gateway: " + .metadata.namespace + "/" + .metadata.name'
+    Found managed gateway: default/gateway
+    {{< /text >}}
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/upgrade-notes/index.md:40]:
```suggestion
{{< text bash >}}
$ kubectl get gateways.gateway.networking.k8s.io -ojson | jq -r '.items[] | select(.spec.gatewayClassName == "istio") | select((.spec.addresses | length) == 0) | "Found managed gateway: " + .metadata.namespace + "/" + .metadata.name'
Found managed gateway: default/gateway
{{< /text >}}
```

File: content/zh/news/releases/1.18.x/announcing-1.18/upgrade-notes/index.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+如果您看到“Found managed gateway”信息，则您可能会受到此变更的影响。
+
+在 Istio 1.18 版之前，托管网关通过创建一个最小的 Deployment
+配置的形式工作，该 Deployment 在运行时通过 Pod 注入进行完全填充。
+如果要升级网关，用户需要重启 Pod 来触发重新的注入操作。
+
+在 Istio 1.18 版中，该行为已变更为创建一个不再依赖于注入的完整 Deployment。
+因此，*网关将在其修订版本变更时通过滚动重启进行更新*。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18/upgrade-notes/index.md:49]:
```suggestion
因此，**网关将在其修订版本变更时通过滚动重启进行更新**。
```
中文可以改成粗体

**Review by windsonsea — COMMENTED** [review]
Thanks. Overall good job!

**Comment by wilsonwu** [comment]
> Thanks. Overall good job!

Thanks for the carefully review, it is a timely work for us :)

**Review by hanxiaop — APPROVED** [review]
Nice!

**Review by windsonsea — APPROVED** [review]
Thanks. Great Job!

**Pull request merged**